.env.tpl

```diff
@@ -65,3 +65,10 @@ ZAC_URL=op://Dimpact/zaakafhandelcomponent-zac-dev/website
 DOCKER_COMPOSE_LDAP_TEST_USER_1_EMAIL_ADDRESS=op://Dimpact/ZAC-.env-$APP_ENV/DOCKER_COMPOSE/LDAP_TEST_USER_1_EMAIL_ADDRESS
 DOCKER_COMPOSE_LDAP_TEST_USER_2_EMAIL_ADDRESS=op://Dimpact/ZAC-.env-$APP_ENV/DOCKER_COMPOSE/LDAP_TEST_USER_2_EMAIL_ADDRESS
 DOCKER_COMPOSE_LDAP_GROUP_A_EMAIL_ADDRESS=op://Dimpact/ZAC-.env-$APP_ENV/DOCKER_COMPOSE/LDAP_GROUP_A_EMAIL_ADDRESS
+
+JAVA_TOOL_OPTIONS="-javaagent:scripts/OpenTelemetry/opentelemetry-javaagent.jar"
```

Comment:
This only works when you run ZAC in IntelliJ I guess? Because when you run ZAC in Docker the JAR is somewhere else. How does the latter work? Which JAVA_TOOL_OPTIONS takes precedence?

Reply:
This is for local. For docker it is included in the Containerfile